networkx/algorithms/approximation/tests/test_treewidth.py

```diff
@@ -132,13 +132,16 @@ def test_empty_graph(self):
         _, _ = treewidth_min_degree(G)
 
     def test_two_component_graph(self):
-        """Test empty graph"""
         G = nx.Graph()
         G.add_node(1)
         G.add_node(2)
         treewidth, _ = treewidth_min_degree(G)
         assert treewidth == 0
 
+    def test_not_sortable_nodes(self):
+        G = G = nx.Graph([(0, "a")])
+        treewidth_min_degree(G)
+
```

Comment:
```suggestion
    def test_not_sortable_nodes(self):
        G = nx.Graph([(0, "a")])
        treewidth_min_degree(G)

```
In principle we could add a docstring to the test as well (i.e. should not raise even when nodes are not sortable), but the test name is probably sufficient!